### PR TITLE
Category

### PR DESCRIPTION
**Pull request by IhorKalaur**

### CODE REVIEW

File: src/main/java/com/example/bookstore/exceptions/GlobalExceptionHandler.java [hunk 1]
```diff
@@ -49,6 +49,19 @@ protected ResponseEntity<Object> handleRegistrationException(
...
     }
 
+    @ExceptionHandler(EntityNotFoundException.class)
+    protected ResponseEntity<Object> handleEntityNotFoundException(
+            EntityNotFoundException ex,
+            WebRequest request) {
+        ErrorResponseDto errorResponse = new ErrorResponseDto(
+                LocalDateTime.now(),
+                HttpStatus.NOT_FOUND,
+                "EntityNotFoundException",
```
Comment (vovasalyha) [inline src/main/java/com/example/bookstore/exceptions/GlobalExceptionHandler.java]: IMO exposing your internal exception to the user is redundant from different aspects

File: src/main/resources/db/changelog/changes/08-create-books-categories-table.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users_roles-table
+      author: ihor-kalaur
+      changes:
+        - createTable:
+            tableName: books_categories
+            columns:
```
Comment (vovasalyha) [inline src/main/resources/db/changelog/changes/08-create-books-categories-table.yaml:8]: add also constraint primary key (book_id, category_id)

**Review by vovasalyha — CHANGES_REQUESTED** [review]
GJ. Let's improve it a bit

**Review by andrii-hoienko — APPROVED** [review]
GJ

**Pull request merged**